docs/Deprecating-Disabling-and-Removing-Casks.md

```diff
@@ -0,0 +1,110 @@
+# Deprecating, Disabling and Removing Casks
+
+There are many reasons why casks may be deprecated, disabled or removed. This document explains the differences between each method as well as explaining when one method should be used over another.
+
+## Overview
+
+These general rules of thumb can be followed:
+
+- `deprecate!` should be used for casks that _should_ no longer be used.
+- `disable!` should be used for casks that _cannot_ be used.
+- Casks that are no longer acceptable in `homebrew/cask` or have been disabled for over a year _should_ be removed.
```

Comment:
> Use it as a temporary way to still serve the binary during the deprecation period. It would allow people to `brew extract` the final state of the cask and put it in their own tap if they really want to.

Makes sense to me 👍🏻

Reply:
@p-linnane Can you make a wording suggestion here, I don't know what's being suggested here really.

Reply:
No wording changes needed. Just wanted us to be on the same page about using archive.org links when we deprecate casks. We seem to be in agreement.